lib/src/Cookie.cc

```diff
@@ -29,9 +29,9 @@ std::string Cookie::cookieString() const
             .append(utils::getHttpFullDate(expiresDate_))
             .append("; ");
     }
-    if (!maxAge_.empty())
+    if (maxAge_ > 0)
```

Comment:
So from MDN: Max-Age Indicates the number of seconds until the cookie expires. A zero or negative number will expire the cookie immediately.
So I think its preferable to have maxAge_ >= 0.
Since int doesn't have a null or empty like string, then any negative number can be considered as not to include.

Reply:
We could use optional for empty value.

Reply:
That works great! Good to know about optional now.